dashboard/client/src/components/EventTableMockData.ts

```diff
@@ -0,0 +1,99 @@
+// eslint-disable-next-line prefer-arrow/prefer-arrow-functions
+
+const SEVERITIES = ["INFO", "DEBUG"];
+const SOURCES = ["Serve", "GCS", "AUTOSCALER"];
+const MESSAGES = [
+  "Serve App {app name} started. Serve App {app name} started. Serve App {app name} started.  {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started. ",
+  "Serve App {app name} deleted. {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Deployment {deployment name} created {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Deployment {deployment name} is running {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Deployment {deployment name} deleted. {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Serve Deployment “job” started. {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Replica {replica name} started {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Replica {replica name} died",
+  "Ray Serve server is running.",
+  "Ray Serve server is terminated.",
+  "Ray Serve HTTP Proxy started on node {node id}, {node ip}",
+  "Ray Serve HTTP Proxy removed from node {node id}, {node ip}",
+];
+
+let longMessage = "";
+const targetLength = 500;
+
+while (longMessage.length < targetLength) {
```

Comment:
avoid mutable variables, especially at top-level scope.

one trick you can use is create a function and call it right away:
```
const longMessage = (() => { ...logic })();
```

Reply:
SG! IIFE is better to be used here